ppr-ui/tests/unit/RegistrationLengthTrust.spec.ts

```diff
@@ -121,7 +121,6 @@ describe('RegistrationLengthTrust RL tests', () => {
     expect(wrapper.vm.showErrorLienAmount).toBe(false)
     expect(wrapper.vm.showErrorSurrenderDate).toBe(false)
     expect(wrapper.vm.minSurrenderDate).toBeDefined()
-    expect(wrapper.vm.maxSurrenderDate).toBeDefined()
```

Comment:
I wonder if we should pop in a little surrender date validation test while we are in here

Reply:
Unit test added in second commit.

Reply:
I was thinking something like 
wrapper.vm.$data.minSurrenderDate= '2012-08-01'
expect(wrapper.vm.lienAmountMessage).toBe('Bad date')

if you know what I mean... not to be picky :)